src/cplscheme/SerialCouplingScheme.hpp

```diff
@@ -84,5 +88,27 @@ class SerialCouplingScheme : public BiCouplingScheme {
   void exchangeInitialData() override;
 };
 
+/*
+ * @brief A fixture that is used to access private functions of the SerialCouplingScheme class.
+ *
+ * The fixture can be used to call private functions for individual testing. 
+ */
+struct serialCouplingSchemeFixture {
+  bool isImplicitCouplingScheme(SerialCouplingScheme &cplscheme)
+  {
+    return cplscheme.isImplicitCouplingScheme();
+  }
+
+  CouplingData *getReceiveData(SerialCouplingScheme &cplscheme, int dataID)
+  {
+    return cplscheme.getReceiveData(dataID);
+  }
+
+  CouplingData *getSendData(SerialCouplingScheme &cplscheme, int dataID)
+  {
+    return cplscheme.getSendData(dataID);
+  }
+};
+
```

Comment:
Could you move this into a separate header in `src/testing/` and into the `precice::testing` namespace?
Also we use capitalized names for types `struct SerialCouplingSchemeFixture`.

Reply:
> Also we use capitalized names for types `struct SerialCouplingSchemeFixture`.

Then our format checker should also check this. I think the same issue is present in #921. Just added a quick review over there.

I will quickly take care of the rest.

Reply:
Done in d6342d3 and deae723